pkg/builder/builder.go

```diff
@@ -103,11 +103,27 @@ func BuildHlSvc(nm types.NamespacedName, vdb *vapi.VerticaDB) *corev1.Service {
 	}
 }
 
+// buildConfigVolumeMount returns the volume mount for config.
+// If vclusterops flag is enabled we mount only /opt/vertica/config/https_certs
+func buildConfigVolumeMount(vdb *vapi.VerticaDB) corev1.VolumeMount {
+	subPath := "config"
```

Comment:
I didn't think I provided enough rationale for this change. Can you include a comment? Something like this:
```
// When using vclusterOps, we don't have any state in /opt/vertica/config to preserve. There is no
// equivalent file like admintools.conf. The only state we need to preserve is the the contents of
// the https_certs directory.
```